components/Banners/TopBanner.jsx

```diff
@@ -0,0 +1,23 @@
+import Button from '../Button/Button'
```

Comment:
używaj albo importów relatywnych albo aliasów. Jeżeli używasz aliasów jak na app page, to importy relatywne powinny być tylko dla elementów, które zawsze pozostają w takiej samej relacji (jak style, podkomponenty itp)